docs/geedocs/5.3.3/answer/fusionAdministration/confTaskRulesForFusionPerf.html

```diff
@@ -495,4 +495,4 @@ <h3>Navigation</h3>
       Created using <a href="http://sphinx-doc.org/">Sphinx</a> 1.7.5.
```

Comment:
We should back out these 5.3.3 changes fully if we can. It doesn't look like these changes matter, but we shouldn't be changing any 5.3.3 docs anyway.

I may be able to do that from the web interface.

Reply:
Oops, I had misconfigured something in Emacs and it added newlines at the end of these when I backed out the other changes...